src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> read(Path path) {
+        List<String> strings;
+        try {
+            strings = Files.readAllLines(path);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file");
+        }
+        return strings;
+    }
```

Comment:
```suggestion
    public List<String> read(Path path) {
        try {
            return Files.readAllLines(path);
        } catch (IOException e) {
            throw new RuntimeException("Can't read file from: " + path.toString(), e);
        }
    }
```

Reply:
strive to provide additional info about exception and always pass exception object (e) into throw new statement